### PR TITLE
Pass data series options through to superclass' constructor

### PR DESCRIPTION
**Pull request by nawatts**

### CODE REVIEW

**Review by nickdoesdesign — APPROVED** [review]
Looks good.

File: README.MD [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 ### Development
 
-_(Make sure to install [https://nodejs.org/download/](Node.js) and run `npm install`)_
+_(Make sure to install [Node.js](https://nodejs.org) and run `npm install`)_
```
Comment (rcreasi) [inline README.MD:5]: I guess nobody ever tried clicking these links 😆

**Comment by rcreasi** [comment]
:+1:

**Review by rcreasi — APPROVED** [review]

**Comment by choochburger** [comment]
Hot stuff 🔥  👍

**Pull request merged**